pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.28</version>
+        <version>3.39</version>
```

Comment:
3.40-20190308.114139-1 from https://github.com/jenkinsci/plugin-pom/pull/173 should fix the Javadoc crash

Reply:
Ok, I'll pick that up. Not sure why it was rejected, maybe because I have write access to the repo and you don't? I usually leave the box that allows people to push to your branch checked.

Reply:
Done in 6ac1d1b.